lib/commands.py

```diff
@@ -893,10 +919,12 @@ def add_global_options(parser):
     group.add_argument("-v", "--verbose", action="store_true", dest="verbose", default=False, help="Show debugging information")
     group.add_argument("-D", "--dir", dest="electron_cash_path", help="electron cash directory")
     group.add_argument("-P", "--portable", action="store_true", dest="portable", default=False, help="Use local 'electron_cash_data' directory")
-    group.add_argument("-w", "--wallet", dest="wallet_path", help="wallet path")
```

Comment:
Why do this? Does this create much benefit?

Reply:
Moved it to separate function because if wallet path is needed in commands, it should be added(instead of all the global arguments added), it is the same as offline argument for restore.

Reply:
@cculianu